apps/condo/domains/ticket/tasks/reopenDeferredTickets.js

```diff
@@ -38,13 +41,37 @@ const reopenDeferredTickets = async () => {
         // TODO(DOMA-4155): Update ticket can break on getOrCreateContactByClientData
         for (const ticket of ticketsToChange) {
             try {
-                await Ticket.update(adminContext, ticket.id, {
+                const updatedData = {
                     dv: 1,
                     sender: { fingerprint: 'auto-reopen', dv: 1 },
-                    executor: { disconnectAll: true },
-                    assignee: { disconnectAll: true },
                     status: { connect: { id: STATUS_IDS.OPEN } },
-                })
+                }
+                const assigneeId = get(ticket, ['assignee', 'id'])
+                const executorId = get(ticket, ['executor', 'id'])
+                const organizationId = get(ticket, ['organization', 'id'])
+                if (assigneeId) {
+                    const employeeCount = await OrganizationEmployee.count(keystone, {
+                        user: { id: assigneeId },
+                        organization: { id: organizationId },
+                        isBlocked: false,
+                        deletedAt: null,
+                    })
+                    if (!employeeCount) {
+                        updatedData.assignee = { disconnectAll: true }
+                    }
+                }
+                if (executorId) {
+                    const employeeCount = await OrganizationEmployee.count(keystone, {
+                        user: { id: executorId },
+                        organization: { id: organizationId },
+                        isBlocked: false,
+                        deletedAt: null,
+                    })
+                    if (!employeeCount) {
+                        updatedData.executor = { disconnectAll: true }
+                    }
+                }
+                await Ticket.update(adminContext, ticket.id, updatedData)
```

Comment:
Why you need an admin context? It’s already inside server utils

Reply:
it's a task, and Alex just named it so, actually it's just `context`

Reply:
deleted skip access and renamed `adminContext` to `context`